conans/model/build_info.py

```diff
@@ -742,7 +780,7 @@ def deduce_full_cpp_info(self, conanfile):
             common.type = str(PackageType.HEADER)  # the type of components is a string!
             common.requires = list(result.components.keys()) + (self.requires or [])
 
-            result.components["_common"] = common
+            # result.components["_common"] = common
```

Comment:
@memsharded Did we need this?

Reply:
Yes, the component must exist, otherwise it won't generate the correct information in generators. I am not sure I understand the question.